Formula/arangodb.rb

```diff
@@ -49,6 +57,8 @@ def install
       system "cmake", "..", *args
       system "make", "install"
 
+      mv "#{buildpath}/usr/bin/arangodb-darwin-amd64", "#{bin}/arangodb"
```

Comment:
We build from source, and do not install binaries.

Reply:
Hi @fxcoudert, the ArangoDB Starter is a separate github project, which published a single binary used by ArangoDB.

How should this be implemented? Do I need to create another formular and require it here? Or load the source as resource and build it?

Reply:
> load the source as resource and build it

That. But it is not clear why we want it as part of the main formula, if it is not shipped by default by upstream.

Reply:
The RedHat and Debian packages contain the ArangoDB Starter as well. The load the static go binary when creating the packages.

Reply:
@fxcoudert I change the formular to build the starter from scratch.